core/templates/components/forms/schema-based-editors/schema-based-editor.directive.ts

```diff
@@ -99,7 +99,7 @@ export const ScopeProvider = {
 export class SchemaBasedEditorDirective extends UpgradeComponent {
   @Input() schema: () => Schema;
   @Input() isDisabled: () => boolean;
-  @Input() localValue;
+  @Input() localValue: string;
```

Comment:
This is not necessarily a string. It's best you use unknown for this. We get a lot of different inputs for this.

Reply:
done